### DISCUSSION TITLE
Show both icon and text for info_entities

### DISCUSSION THREAD

**Discussion by HamsenBender** [marcokreeft87/room-card#220] [Q&A] [answered]
Is there an option to show both the icon and text for the info entities?
When I want to show two values with the same unit for a room, f.e.:
* Humidity: 60%
* Ventilation: 50%

It would be nice to be able to show both icon and text, to indicate what the value represents.

Tx
Jan

**Comment by marcokreeft87** [comment, marked as answer]
There is not room for that. The text would be too small or the icon would be too small. My advice would be moving it to the entities. 

There is not option to do that with info_entities

**Reply by HamsenBender** [reply]
That's indeed a good idea, I'll move the entities which can be controlled to the entities section.
Tx!
